oscps-lib/src/component.rs

```diff
@@ -59,28 +60,64 @@ impl Chemical {
     }
 }
 
-struct ChemicalProperties {
-    pub melting_pt : Option<uom::si::f64::ThermodynamicTemperature>,
-    pub boiling_pt : Option<uom::si::f64::ThermodynamicTemperature>,
-    pub density : Option<uom::si::f64::MassDensity>,
-    pub molec_mass : Option<uom::si::f64::Mass>
+pub struct ChemicalProperties {
```

Comment:
I do think we should use uom down the line, but if this was needed to get it to work, that's fine for now.